build/windows/Makefile.ps1

```diff
@@ -161,13 +161,51 @@ if ($isCDPxEnvironment) {
   Write-Host("successfully got latest go modules") -ForegroundColor Green
 
   go build -ldflags "-X 'main.revision=$buildVersionString' -X 'main.builddate=$buildVersionDate'" -buildmode=c-shared -o out_oms.so .
+
+  $inputdir = Join-Path -Path $rootdir -ChildPath "source\plugins\go\input"
+  Set-Location -Path $inputdir
+
+  Write-Host("Installing the cmetrics-go package")
+  git clone https://github.com/calyptia/cmetrics-go.git
```

Comment:
I see the windows support in calyptia upstream not merged. dont we need to merge that PR?

Reply:
No that is not needed...As I have already added the code locally as well for now